stream-chat-android/src/main/java/com/getstream/sdk/chat/adapter/viewholder/message/configurators/MessageTextConfigurator.kt

```diff
@@ -34,7 +34,7 @@ internal class MessageTextConfigurator(
     private val style: MessageListViewStyle,
     private val bubbleHelper: MessageListView.BubbleHelper,
     private val messageClickListener: MessageListView.MessageClickListener,
-    private val messageLongClickListener: MessageListView.MessageLongClickListener,
+    private var messageLongClickListener: MessageListView.MessageLongClickListener,
```

Comment:
Do we need it as mutable field?
It seems that is not been updated

Reply:
No need. This one passed me by.